app/scripts/views/home/home-controller.js

```diff
@@ -6,55 +6,89 @@
      * Controller for the imls app home view
      */
     /* ngInject */
-    function HomeController($log, $q, Geocoder, Museum) {
-
+    function HomeController($log, $q, $geolocation, Geocoder, Museum) {
         var ctl = this;
 
+        var SEARCH_DIST_DECIMAL_DEGREES = 0.1;
+
         initialize();
 
         function initialize() {
-            ctl.error = false;
-            ctl.mapExpanded = false;
+            ctl.list = [];
+            ctl.safeList = [];
+            ctl.states = {
+                DISCOVER: 0,
+                LIST: 1,
+                ERROR: 2
```

Comment:
These constants are for view state, and used to switch which section of the main content is displayed. They're not used to indicate particular errors.